frontend/spa-assemble-config.json

```diff
@@ -36,9 +35,6 @@
     "@openmrs/esm-service-queues-app": "latest",
     "@openmrs/esm-system-admin-app": "latest",
     "@openmrs/esm-user-onboarding-app": "latest",
-    "@openmrs/esm-ward-app": "latest",
-    "@openmrs/esm-stock-management-app": "latest",
-    "@openmrs/esm-billing-app": "latest",
```

Comment:
Are we also removing the backend billing module?

Reply:
Historically, I don't think we have.

Reply:
We should also do so.

Reply:
I think the plan is to get it in the ref app for 3.4.0, because there's some more testing and work left to do

Reply:
Oh... ok.